src/main/java/thePackmaster/patches/OpeningRunScreenPatch.java

```diff
@@ -18,9 +19,17 @@ public class OpeningRunScreenPatch {
     )
     public static void SetTheThing(NeowEvent __instance) {
         if (AbstractDungeon.player.chosenClass == ThePackmaster.Enums.THE_PACKMASTER) {
-            BaseMod.logger.info("Packmaster is Ready To Do Thing");
-            SpireAnniversary5Mod.openedStarterScreen = false;
-            SpireAnniversary5Mod.doPackSetup = true;
+            if (SpireAnniversary5Mod.allPacksMode) {
+                BaseMod.logger.info("Vex's All Packs Override Enabled. Skipping intro screen");
+                SpireAnniversary5Mod.currentPoolPacks.clear();
+                SpireAnniversary5Mod.currentPoolPacks.addAll(SpireAnniversary5Mod.unfilteredAllPacks);
```

Comment:
Should we respect filtering out packs when setting this up? My immediate reaction is to say that we should, so that "all packs mode" means "all packs except ones that have been filtered out". If someone has a pack disabled for normal play, they probably don't want to see it in all pack mode either.

Reply:
Definitely think we should respect the filtering

Reply:
And done! Good call